example/tutorial/tensorflow_example.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from mlflow import tensorflow, tracking
+import numpy as np
+import pandas as pd
+import shutil
+import tempfile
+import tensorflow as tf
+
+
+def main(argv):
+    # Builds, trains and evaluates a tf.estimator. Then, exports it for inference, logs the exported model 
+    # with MLflow, and loads the fitted model back as a PyFunc to make predictions.
+    (x_train, y_train), (x_test, y_test) = tf.keras.datasets.boston_housing.load_data()
+    feat_cols = [tf.feature_column.numeric_column(key="features", shape=(13,))]
```

Comment:
Using these magic #s is scary. Can we figure this out from dataset? The concern is that if the shape of this data changes, this example will break.

Reply:
As per @smurching's comment above, I was able to change the `102` to `None`. The 13 is based off of the number of feature columns used for training/testing/inference.